src/signals/incident-management/index.js

```diff
@@ -70,6 +71,7 @@ export const IncidentManagementModuleComponent = ({
         <Route exact path={routes.incidents} component={IncidentOverviewPage} />
         <Route exact path={routes.incident} component={IncidentDetail} />
         <Route exact path={routes.split} component={LegacyIncidentSplitContainer} />
+        <Route exact path={routes.splitIncident} component={IncidentSplitContainer} />
```

Comment:
Isn't `IncidentSplitContainer` supposed to replace `IncidentSplitContainer`? If so, why is there a need for two routes?

Reply:
Like discussed, it was unclear if the legacy one had to stay, done